packages/cli/src/util/command.ts

```diff
@@ -7,5 +7,6 @@ export function createMedplumCommand(name: string): Command {
     .option('--base-url <baseUrl>', 'FHIR server base url')
     .option('--token-url <tokenUrl>', 'FHIR server token url')
     .option('--authorize-url <authorizeUrl>', 'FHIR server authorize url')
-    .option('--fhir-url-path <fhirUrlPath>', 'FHIR server url path');
+    .option('--fhir-url-path <fhirUrlPath>', 'FHIR server url path')
+    .option('--auth-type <authType>', 'Type of authentication', 'basic');
```

Comment:
Is this where we can list all auth types?  We should determine our list, and the exact strings we want to accept?

We have this enum, which might be a good starting point:

```ts
export enum OAuthGrantType {
  ClientCredentials = 'client_credentials',
  AuthorizationCode = 'authorization_code',
  RefreshToken = 'refresh_token',
  JwtBearer = 'urn:ietf:params:oauth:grant-type:jwt-bearer',
  TokenExchange = 'urn:ietf:params:oauth:grant-type:token-exchange',
}
```

So maybe:

- `basic` - Basic auth
- `client_credentials` - Client credentials flow
- `authorization_code` - Authorization code flow (default)
- `jwt-bearer` - User provided presigned JWT
- `token-exchange` - Preconfigured external auth provider